MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -1743,6 +1743,13 @@ private string GetFixedSizeScalingFilter(Video3DFormat? threedFormat, int reques
 
                 filters.Add(subParam);
 
+		// Ensure proper filters are passed to ffmpeg in case of hardware acceleration via VA-API
+		// Reference: https://trac.ffmpeg.org/wiki/Hardware/VAAPI
+		if (string.Equals(outputVideoCodec, "h264_vaapi", StringComparison.OrdinalIgnoreCase))
```

Comment:
I wonder what happens if a `vp9_vaapi` or `hevc_vaapi` is selected... :)

Reply:
To answer this question, I need to tell why I did that in the first place: I looked at the ffmpeg command line that was failing, and then looked at the code *where* those filters were set up. They were set up only with `h264_vaapi` (but I don't know why in the first place).

I can't unfortunately test with vp9 or hevc as the CPU of the server I run Jellyfin on doesn't support them natively.